frontend/express/public/core/session-overview/javascripts/countly.views.js

```diff
@@ -0,0 +1,97 @@
+/* global countlyVue,CV,countlySessionOverview,app*/
+var SessionOverviewView = countlyVue.views.create({
+    template: CV.T("/core/session-overview/templates/session-overview.html"),
+    data: function() {
+        return {};
+    },
+    computed: {
+        sessionOverview: function() {
+            return this.$store.state.countlySessionOverview.sessionOverview;
+        },
+        isLoading: function() {
+            return this.$store.state.countlySessionOverview.isLoading;
+        },
+        selectedDatePeriod: {
+            get: function() {
+                return this.$store.state.countlySessionOverview.selectedDatePeriod;
+            },
+            set: function(value) {
+                this.$store.dispatch('countlySessionOverview/onSetSelectedDatePeriod', value);
+                this.$store.dispatch('countlySessionOverview/fetchAll');
+            }
+        },
+        sessionOverviewRows: function() {
+            return this.$store.state.countlySessionOverview.sessionOverview.rows;
+        },
+        sessionOverviewOptions: function() {
+            return {
+                xAxis: {
+                    data: this.xAxisSessionOverviewDatePeriods
+                },
+                series: this.yAxisSessionOverviewCountSeries
+            };
+        },
+        xAxisSessionOverviewDatePeriods: function() {
+            return this.$store.state.countlySessionOverview.sessionOverview.rows.map(function(tableRow) {
+                return tableRow.date;
+            });
+        },
+        yAxisSessionOverviewCountSeries: function() {
+            return this.sessionOverview.series.map(function(sessionOverviewSerie) {
+                return {
+                    data: sessionOverviewSerie.data,
+                    name: sessionOverviewSerie.label,
+                };
+            });
+        }
+    },
+    methods: {
+        refresh: function() {
+            this.$store.dispatch('countlySessionOverview/fetchAll');
+        }
+    },
+    mounted: function() {
+        this.$store.dispatch('countlySessionOverview/fetchAll');
+    },
+});
+
+//Note: the parent component that renders all session analytics tabs.
+var SessionAnalyticsView = countlyVue.views.create({
+    template: CV.T("/core/session-overview/templates/session-analytics.html"),
+    mixins: [
+        countlyVue.container.tabsMixin({
+            "sessionAnalyticsTabs": "/analytics/sessions"
+        })
+    ].concat(countlyVue.container.mixins(["/analytics/sessions"])),
```

Comment:
i couldn't find anywhere where you are registering a mixin. like [this ](https://github.com/Countly/countly-server/blob/3b0bc677e21f68bbaeed49064352f40b8776211f/plugins/vue-example/frontend/public/javascripts/countly.views.js#L485-L498).
countlyVue.container.mixins fetches all the mixins from other views, but i dont think thats the case here. Is it?

Reply:
You're right, I forgot to remove it as I was trying to understand and play around with container mixins